drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -449,14 +547,6 @@ static int stm32_clock_control_init(struct device *dev)
 
 #endif /* CLOCK_STM32_SYSCLK_SRC */
 
-	/* Set buses (Sys,AHB, APB1, APB2 & APB4) prescalers */
-	LL_RCC_SetSysPrescaler(sysclk_prescaler(CONFIG_CLOCK_STM32_D1CPRE));
-	LL_RCC_SetAHBPrescaler(ahb_prescaler(CONFIG_CLOCK_STM32_HPRE));
-	LL_RCC_SetAPB1Prescaler(apb1_prescaler(CONFIG_CLOCK_STM32_D2PPRE1));
-	LL_RCC_SetAPB2Prescaler(apb2_prescaler(CONFIG_CLOCK_STM32_D2PPRE2));
-	LL_RCC_SetAPB3Prescaler(apb3_prescaler(CONFIG_CLOCK_STM32_D1PPRE));
-	LL_RCC_SetAPB4Prescaler(apb4_prescaler(CONFIG_CLOCK_STM32_D3PPRE));
-
 	/* Set FLASH latency */
 	/* AXI clock is SYSCLK / HPRE */
 	LL_SetFlashLatency(get_bus_clock(CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC,
```

Comment:
There is a potential issue with Flash latency: 
When clock configuration increases Flash frequency (=AXI clock = HCLK), latency should be set first.
When frequency is decreased, latency should be set after clock configuration.
Issue is probably not visible because after reset default latency is set the max value.
You can check the way it is done for other series in 'clock_stm32_ll_common.c'

Reply:
Exactly, in this case we are only tweaking down the flash latency as reset configuration sets it to the maximum value for H7 series (7 wait states).

So it cannot be an issue, can it ?

Reply:
It is not an issue today, ... but in the future ...
Nevertheless, I would prefer to keep coherence with other series which are tackling with this.

Reply:
Alright then 👍 , seeing the common stm32 implementation it should be implementable in this PR.
I'll do my best to implement the method correctly (but I bet it won't change code execution in the end).

Reply:
> (but I bet it won't change code execution in the end)
You're doing it for generations after you LOL

Reply:
> > (but I bet it won't change code execution in the end)
> 
> You're doing it for generations after you LOL

Yes that's why you changed my mind, I'm preparing something clean for the future generation 👍

Reply:
@Nukersson @ABOSTM check my latest commit to see if this new flash latency setting implementation suits you.

Reply:
> @Nukersson @ABOSTM check my latest commit to see if this new flash latency setting implementation suits you.

Checked it currently out. Still works fine for me with `help_world` and M7 core on my nucleo-h745zi-q board. I've not tested the flash latency properly. I will just hope, it works fine.